en/reference/mtls.md

```diff
@@ -0,0 +1,214 @@
+---
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Mutually authenticated TLS (mTLS) reference"
+---
+
+{% include note.html content="See [Securing Vespa with mutually authenticated TLS (mTLS)](../mtls.html) 
+on how to secure Vespa with mutually authenticated TLS."
+%}
+
+## Environment variables
+
+
+<table class="table table-striped">
+    <tr><th scope="col" class="col-xs-3">Name</th><th scope="col" class="col-xs-9">Description</th></tr>
+    <tr>
+        <td>VESPA_TLS_CONFIG_FILE</td>
+        <td>Absolute path JSON configuration file with TLS configuration.</td>
+    </tr>
+    <tr>
+        <td>VESPA_TLS_INSECURE_MIXED_MODE</td>
+        <td markdown="span">Enables TLS mixed mode. See [TLS Mixed mode](#tls-mixed-mode) for possible values.</td>
+    </tr>
+</table>
+
+### <a name="tls-mixed-mode"/>TLS mixed mode
+Possible TLS mixed mode settings for `VESPA_TLS_INSECURE_MIXED_MODE`:
+
+<table class="table table-striped">
+    <tr><th scope="col" class="col-xs-3">Name</th><th scope="col" class="col-xs-9">Description</th></tr>
+    <tr>
+        <td>plaintext_client_mixed_server</td>
+        <td>Clients do not use TLS, servers accept both TLS and plaintext clients.</td>
+    </tr>
+    <tr>
+        <td>tls_client_mixed_server</td>
+        <td>Clients use TLS, servers accept both TLS and plaintext clients.</td>
+    </tr>
+    <tr>
+        <td>tls_client_tls_server</td>
+        <td>All clients and servers use TLS only.</td>
+    </tr>
+</table>
+
+### Configuration file
+The TLS configuration file contains a single top-level JSON object.
+
+#### <a name="top-level-elements"/>Top-level elements
+<table class="table table-striped">
+    <tr><th scope="col" class="col-xs-2">Name</th><th scope="col" class="col-xs-1">Required</th><th scope="col" class="col-xs-9">Description</th></tr>
+    <tr>
+        <td markdown="span">[files](#files)</td>
+        <td>Yes</td>
+        <td>JSON object containing file system paths crypto material.</td>
+    </tr>
+    <tr>
+        <td>accepted-ciphers</td>
+        <td>No</td>
+        <td>JSON array of accepted TLS cipher suites.</td>
+    </tr>
+    <tr>
+        <td>accepted-protocols</td>
+        <td>No</td>
+        <td>JSON array of accepted TLS protocol versions.</td>
```

Comment:
Add link to section for defaults for both `accepted-ciphers` and `accepted-protocols`